invenio_cli/commands/translations.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2022 CERN.
+#
+# Invenio-Cli is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Invenio module to ease the creation and management of applications."""
+
+from pathlib import Path
+
+from ..commands import Commands
+from .steps import CommandStep
+
+
+class TranslationsCommands(Commands):
+    """Translations CLI commands."""
+
+    CMD_PREFIX = ["pipenv", "run"]
```

Comment:
This is a common pitfall across the whole library. It should potentially be moved up to the `Commands` class, or to the command class since `pipenv run...` is a common prefix for all `CommandSteps`.